x-pack/plugins/rule_registry/common/field_map/types.ts

```diff
@@ -12,5 +12,6 @@ export interface FieldMap {
     array?: boolean;
     path?: string;
     scaling_factor?: number;
+    dynamic?: 'strict' | boolean;
```

Comment:
this change is no longer needed right?

Reply:
It's still needed to set `dynamic: true` on the field mapping for the reason [stated above](https://github.com/elastic/kibana/pull/144484#discussion_r1014857283)